app/facades/back_end_facade.rb

```diff
@@ -60,5 +74,45 @@ def get_user_events(user_id)
 
       events[:data].map { |event| UserEvent.new(event) }
     end
+
+    def get_selected_gym(yelp_gym_id)
+      gym = BackEndService.get_one_gym(yelp_gym_id)
+      return unless gym[:data]
+
+      YelpGym.new(gym[:data])
+    end
+
+    def get_gym_users(yelp_api_key)
+      users = BackEndService.get_gym_users(yelp_api_key)
+
+      if users[:data]
+        users[:data].map do |user|
+          GymUser.new(user)
```

Comment:
I would suggest updating the PORO to just be user - but we can come back to for timesake if needed